types/theme-ui/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "version": "0.2.0",
+  "name": "@types/theme-ui",
+  "private": true,
+  "dependencies": {
+    "@emotion/core": "^10.0.14",
+    "@types/styled-system__css": "^5.0.0",
```

Comment:
Think this should be `styled-system/css`? You can reference it in the `path` of `tsconfig` to make sure the correct and up to date package is used.